lib/state/contest_game_utils.dart

```diff
@@ -0,0 +1,173 @@
+import 'dart:convert';
+import 'package:built_value/standard_json_plugin.dart';
+import 'package:data/data.dart';
+import 'package:flutter/widgets.dart';
+import 'package:jamaica/state/state_container.dart';
+
+contestPerformance(
```

Comment:
Again asking you - what is this method doing. This of methods as an action or a verb. What is the action here?